core/src/main/res/values/styles.xml

```diff
@@ -128,6 +128,28 @@
         <item name="android:fitsSystemWindows">true</item>
     </style>
 
+    <style name="Theme.Base.AntennaPod.Light.Translucent.Dialog" parent="Theme.AntennaPod.Light.Translucent.Dialog">
```

Comment:
Do we really need a new theme here? I think just using the already existing `Theme.AntennaPod.XXX.Translucent` should work. The style is used for an Activity, after all. (also, the `parent` attribute does not reference an existing theme)

Reply:
It works, but it looks a little off to me in this use case, I used it once and I though it was better to create a new theme than changing one already existing, which would impact other screen in the app, I recorded the two implementations (existing theme and newly created theme), just take a look at the status bar and the opening and closing animations with the existing one. (I'm sorry about the quality of the recording, ROM issue)

https://user-images.githubusercontent.com/63880378/158082038-c65d08b5-9a2f-4f8e-bdda-5968d21f723d.mp4


https://user-images.githubusercontent.com/63880378/158082170-2a02824c-cc56-4576-a0df-56f4173e3b56.mp4

Reply:
Then the other transparent theme should be improved instead of creating a new one that does not derive from the standard Material theme. Using a new theme without a valid parent will likely cause weird behavior on some devices. In the past, the AppCompat library crashed when it was used with a theme that does not derive from AppCompat.

Reply:
Are you still interested in this PR, @nicoolasj? If so, please have a look at the comment above :)

Reply:
@ByteHamster  Yes I'm keen on solving this, I was busy with university exams but now I'm free to work on this PR. I'm sorry for the delay, I'm working on this right now, soon I'll push changes and reply your comments above.

Reply:
I updated the theme to reference the existing `Theme.AntennaPod.XXX.NoTitle`, I also used the existing `Theme.AntennaPod.XXX.Translucent` before, but after testing I saw the first one was working better as a valid parent.

Reply:
Hmmm, why that? I think the translucent theme should actually do exactly what we need here. If it doesn't, the translucent theme should be adapted instead of adding a new theme.

Reply:
Got it! I adapted and deleted the new theme I created. I tested the changes in api 19, 21 and 30. Everything seems ok in my testings.